tests/waitForReplica.test.js

```diff
@@ -0,0 +1,75 @@
+import { checkWaitRetry } from "../.reaction/waitForReplica";
+
+describe("waitForReplica.checkWaitRetry", () => {
+  it("should work in immediate success case", async () => {
```

Comment:
We use `test` instead of `it`, and we don't wrap in a `describe` block if there's just one in the file. The rest looks fine. More details here: https://docs.reactioncommerce.com/docs/writing-jest-unit-tests

Reply:
Oh that's really nice. Hadn't found that guide yet, was basically assuming a mocha-compatible API layer. Fixed.